accessibility/house-style.md

```diff
@@ -0,0 +1,125 @@
+# Accessibility style guide
+
+This document outlines the way we make our sites accessible. It's a living styleguide – it will grow as our practices do.
+
+- [General Principles](#general-principles)
+  - [What is accessibility?](#what-is-accessibility)
+  - [Why do we care about accessibility?](#why-do-we-care-about-accessibility)
+- [VPATs](#vpats)
+- [How we conform](#how-we-conform)
+  - [Standards](#standards-we-conform-to)
+  - [Tools and techniques](#tools-and-techniques)
+    - [Guidelines](#guidelines)
+    - [Pa11y](#pa11y)
+    - [Other accessibility testing tools](#other-accessibility-testing-tools)
+    - [Assistive technology](#assistive-technology)
+    - [Testing with real users](#testing-with-real-users)
+    - [Resources](#resources)
+
+
+## General principles
+
+> The power of the Web is in its universality.
+> Access by everyone regardless of disability is an essential aspect.
+>
+> _Tim Berners-Lee, inventor of the World Wide Web_
+
+
+### What is accessibility?
+
+The Web is fundamentally designed to work for all people, whatever their hardware, software, language, culture, location, or physical or mental ability. When the Web meets this goal, it is accessible to people with a diverse range of hearing, movement, sight, and cognitive ability.
+
+Thus the impact of disability is radically changed on the Web because the Web removes barriers to communication and interaction that many people face in the physical world. However, when websites, web technologies, or web tools are badly designed, they can create barriers that exclude people from using the Web.
+
+It is essential that the Web be accessible in order to provide equal access and equal opportunity to people with diverse abilities. Indeed, the [UN Convention on the Rights of Persons with Disabilities](https://www.un.org/development/desa/disabilities/) recognises access to information and communications technologies, including the Web, as a basic human right.
+
+Accessibility supports social inclusion for people with disabilities as well as others, such as older people, people in rural areas, and people in developing countries.
+
+
+### Why do we care about accessibility?
+
+- *It's a legal requirement:* we're legally obliged to make our content accessible to users with diverse needs. All of our websites MUST comply with [US Section 508](https://www.section508.gov/) of the US Rehabilitation Act 1973, [EU Directive 2016/2102](http://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A32016L2102) and the [UK Equality Act 2010](http://www.legislation.gov.uk/ukpga/2010/15/contents). 
```

Comment:
I mention WCAG 2 and the 508 refresh briefly further down in the doc when talking about how we conform. I can't figure out a way to crowbar it into this particular section without making it mostly about the American guidelines, and I want to avoid that so people don't go "oh well that's just America, our product is only sold in $OTHERCOUNTRY". 

You've asked for changes to the "how we conform" section, so how about I make those changes and then expand the single sentence that's left to explain the relationship between 508 and WCAG 2?

Reply:
Yep, sounds good, thanks. Chuck the links in somewhere.